app/src/main/java/com/celzero/bravedns/ui/SettingsFragment.kt

```diff
@@ -675,11 +758,11 @@ class SettingsFragment : Fragment(R.layout.activity_settings_screen) {
                     //creating json object
```

Comment:
such code comments may not be required: https://blog.jim-nielsen.com/2021/useful-and-usless-code-comments/

Reply:
Removed the code-comment.